src/test/java/org/openstreetmap/atlas/utilities/caching/CachingTests.java

```diff
@@ -78,6 +80,10 @@ private void compareFetchedResource(final URI resourceURI, final Resource golden
             .get("src/test/resources/org/openstreetmap/atlas/utilities/configuration/feature.json")
             .toAbsolutePath();
     private static final URI LOCAL_TEST_FILE_2_URI = LOCAL_TEST_FILE_2.toUri();
+    private static final Path LOCAL_TEST_FILE_3 = Paths
+            .get("src/test/resources/org/openstreetmap/atlas/utilities/caching/fileNoExt")
```

Comment:
It might not be the right place to say this, but, it would be better to not have hardcoded paths there (in case we change package names in the future, this _will_ be missed).

How about doing this in the tests below:
```
ByteArrayResource originalFileBytes = new ByteArrayResource();
originalFileBytes.copyFrom(new InputStreamresource(() ->
    CachingTests.class.getResourceAsStream("fileNoExt")));
```

Reply:
This is a good call. I think there are a lot of places in the codebase that could use this change.